components/automate-ui/src/app/helpers/jwt/jwt.ts

```diff
@@ -57,7 +57,7 @@ export class Jwt {
   static parseLicenseToken(token: string): LicenseToken | null {
     let tok: LicenseToken;
     try {
-      tok = jwtDecode(token);
+      tok = jwt_decode(token);
```

Comment:
Is this change still causing errors to be reported by Wallaby?

Reply:
yeah it is, I meant to send them a note last week but didn't.  I'll get one out to them today.

Reply:
The wallaby errors have since disappeared 🤷